try{
    console.log("hello")
}
catch(err){
    console.log("it didnt ma= ")
}
